src/app/organizations/organizations/organizations.component.html

```diff
@@ -0,0 +1,61 @@
+<!--
+   Copyright 2019 OICR
+ *
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ *
+       http://www.apache.org/licenses/LICENSE-2.0
+ *
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<app-header>
+  <mat-icon>people</mat-icon>
+  Available Organizations
+</app-header>
+<div *ngIf="!(loading$ | async)" class="container">
+  <button class="pull-right" color="primary" mat-raised-button (click)="createOrganization()">Create Organization</button>
+  <div *ngIf="(filteredOrganizations$ | async) as organizations ; else noOrganizationsFound">
+    <form [formGroup]="organizationSearchForm">
+      <mat-form-field>
+        <input matInput placeholder="Search organizations" formControlName="name">
+      </mat-form-field>
+    </form>
+    <!-- Far too many organizations async pipe, think of better solution -->
+    <div *ngIf="organizations.length > 0; else noOrganizationsFound">
+      <div fxLayoutAlign="start stretch" fxFlexFill fxLayout="row wrap" fxLayoutGap="10px">
+        <mat-card fxFlex *ngFor="let org of organizations" class="my-2">
+          <mat-card-header>
+            <mat-card-title>
+              <a [routerLink]="org.name">{{org.name}}</a>
+            </mat-card-title>
+            <mat-card-subtitle>{{ org.description }}</mat-card-subtitle>
+          </mat-card-header>
+          <mat-card-content>
+            <div fxLayout="row" fxLayoutGap="10px">
+              <span *ngIf="org?.email">
+                <mat-icon>email</mat-icon>
+                <a [href]="'mailto:' + org.email" target="_top">{{ org.email }}</a>
```

Comment:
I wonder if we should redact the visible email, although that wouldn't really stop a bot. I haven't got there yet in the review, so maybe you already handle it, but if we're displaying a user's email, we should make clear in the UI that if the user specifies an email, it will be made public.

Reply:
I think this is the organization's email https://github.com/broadinstitute 
Worth a warning when creating the org, but probably not an issue for bots

Reply:
Hmm...I'll make that a material form field input hint in the 2nd PR.